### PR TITLE
Add a button to clear radio button selection. 

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**
Screenshot_1591082682

JIRA: https://dimagi-dev.atlassian.net/browse/SAAS-10936

Product Note: Added a button to clear choices in single-select questions.

### CODE REVIEW

**Comment by orangejenny** [comment]
Marvelous! Screenshot?

File: app/src/org/commcare/views/widgets/SelectOneWidget.java [hunk 1]
```diff
@@ -101,17 +107,45 @@ public SelectOneWidget(Context context, FormEntryPrompt prompt) {
                     addView(divider);
                 }
             }
+            addClearButton(context, s != null && !prompt.isReadOnly());
         }
     }
 
+    private void addClearButton(Context context, boolean show) {
+        clearButton = new Button(context);
```
Comment (shubham1g5) [inline app/src/org/commcare/views/widgets/SelectOneWidget.java:115]: would be good to abstract this code into xml. We do have a ton of layout code in Java but it's preferrable to not add onto that mess.

**Comment by shubham1g5** [comment]
@ShivamPokhriyal GTM given build failures are resolved. Would you also be able to add a product note in PR. Thanks!

**Comment by ShivamPokhriyal** [comment]
@damagatchi retest this please

**Pull request merged**